evm/v0.5/contracts/dev/PrepaidAggregator.sol

```diff
@@ -227,6 +234,20 @@ contract PrepaidAggregator is AggregatorInterface, Owned, WithdrawalInterface {
     }
   }
 
+  /**
+   * @notice returns the number of oracles
+   */
+  function oracleCount() public view returns (uint32) {
+    return uint32(oracleAddresses.length);
+  }
+
+  /**
+   * @notice returns an array of addresses containing the oracles on contract
+   */
+  function getOracles() external view returns (address[] memory) {
```

Comment:
This is the only getter in the contract that starts with "get". Maybe `oracleAddresses` or rename oracles with an underscore to follow the private naming convention?

Reply:
There's `getAnswer`, `getTimestamp`, `getTimedOutStatus`, and `getOriginatingRoundOfAnswer` already. The story says use `getOracles`, but I can make the change if you want.